app/src/lib/logging/renderer/install.ts

```diff
@@ -1,20 +1,17 @@
-import { ipcRenderer } from 'electron'
 import { LogLevel } from '../log-level'
 import { formatLogMessage } from '../format-log-message'
+import { sendProxy } from '../../../ui/main-process-proxy'
 
 const g = global as any
+const ipcLog = sendProxy('log')
```

Comment:
Nit: For consistency with other calls like `moveToTrash`, seems like this could be in the `main-process-proxy` and import `ipcLog` from there instead of `sendProxy`.